src/katgpucbf/xbgpu/engine.py

```diff
@@ -157,9 +157,12 @@ class needs to carry a record of which antennas have missed data in the
 
     Parameters
     ----------
-    buffer_device
+    out
         An int8 type :class:`~katsdpsigproc.accel.DeviceArray` with shape
         (frames, ants, channels, spectra_per_frame, N_POLS, COMPLEX).
+    saturated
+        An uint32 type :class:`~aktsdpsigproc.accel.DeviceArray` with shape
```

Comment:
```suggestion
        An uint32 type :class:`~katsdpsigproc.accel.DeviceArray` with shape
```

Reply:
Looks like a failed phishing attempt atm. Thanks for spotting it.